### DISCUSSION TITLE
What happens to mergers and splitters in tobac when no postprocessing is used?

### DISCUSSION THREAD

**Discussion by JuliaKukulies** [tobac-project/tobac#318] [General]
With the release of `tobac v1.5.0` we offer explicit handling of mergers and splitters in the `merge_split.py` module. However, I am curious if you have some insights of what actually happens to `tobac` tracks as they merge and split when no explicit handling is used. 

We have recently looked into what happens with `tobac` tracks that merge and split when the new postprocessing module is NOT used. We found, for example, that in some split cases, `tobac` initializes new tracks instead of continuing the old track. For anybody who knows more about our linking method, do you have ideas why this happens? I would expect that one of the split-off tracks gets assigned the same ID as the before-split track if the propagation speed does not change? 

To give an example: this is an idealized case that initiates with three individual features that all fulfill the criteria for feature detection (a) and move eastward, grow, and merge after eight hours (b). The northern cell splits off after 19 hours (c) by moving towards the northeast (d): 

![idealized_case2](https://github.com/tobac-project/tobac/assets/44163060/38360fbe-4910-46ce-b710-295435e3272f)

`tobac` would identify three individual tracks because when the split-off happens, two new tracks are initiated (the lines refer to the same four timesteps as above): 

![tobac_results](https://github.com/tobac-project/tobac/assets/44163060/b1282445-b743-4a0a-8d0f-28e17f04190d)

**Comment by snilsn** [comment]
I think the problem is that trackpy (which tobac relies on) is not designed for things that merge and split. 

The distance between the last feature of the blue track and the first feature of the green track is quite large, probably larger than the search radius of your linking. So increasing the search radius (e.g. by increasing `v_max` in `linking_trackpy` ) should result in the output you expected, but also in an unrealistic jump in the velocity, since the position changes almost instantaneously from the centre of the three blobs to the centre of the bottom two blobs.

Playing around with the options for `position_threshold` in the feature detection might help alleviate this problem, but I don't think it's completely solvable now.

**Reply by JuliaKukulies** [reply]
Thank you,  @snils, I appreciate your input on this! I was not looking for a solution but mainly wanted to understand what is going on here. What you say about the search radius makes a lot of sense!
